src/lib.rs

```diff
@@ -708,7 +708,10 @@ impl Builder {
 
     /// Sets the target for the log output.
     ///
-    /// Env logger can log to either stdout or stderr. The default is stderr.
+    /// Env logger can log to either stdout, stderr or a custom pipe. The default is stderr.
+    ///
+    /// The custom pipe can be used to send the log to a file directly or something more complex.
+    /// It is advertised to use a wrapper for log files, so that rollover and slow FSs are handled well.
```

Comment:
```suggestion
    /// The custom pipe can be used to send the log messages to a custom sink (for example a file).
    /// Do note that direct writes to a file can become a bottleneck due to IO operation times.
```

I don't think a notice is needed to be honest, but if we add one I'd go for something similar to the above without going into implementation details.